lib/tasks/registration_interest.rake

```diff
@@ -0,0 +1,10 @@
+namespace :registration_interest do
+  desc "Send notifications to users indicating NPQ applications are open"
+  task :send_notifications, %i[count] => :environment do |_t, args|
+    count = args["count"].to_i
+
+    RegistrationInterest.order("RANDOM()").first(count).each do |registration_interest|
```

Comment:
How about we only pull records that have not been notified already?

Reply:
Good spot